Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -229,6 +223,14 @@ var ListView = React.createClass({
      */
     removeClippedSubviews: React.PropTypes.bool,
     /**
+     * Makes the sections headers sticky.  The sticky behavior means that it
```

Comment:
Looking through the files in `Library/CustomCompnents` there are lots of instances of `.<space><space>`. I don't think it's appropriate for this pull request to fix all of them. I will change the 10 instances in the `ListView.js` property comments, but I'll leave the copyright header with the double spaces so it is the same as the other headers in this folder.

It's not a big deal, but I don't want to muddy up this PR with unrelated style changes.